js/web/src/components/Breadcrumbs.js

```diff
@@ -80,15 +80,16 @@ const YourReleasesBreadcrumb = () => {
 }
 
 const routes = [
-  { path: '/', breadcrumb: 'Home' },
+  { path: '/', breadcrumb: () => <Typography variant="subtitle1">Home</Typography> },
   { path: '/releases', breadcrumb: YourReleasesBreadcrumb },
   { path: '/releases/:releasePubkey', breadcrumb: ReleaseBreadcrumb },
-  { path: '/releases/:releasePubkey/market', breadcrumb: 'Market' },
+  { path: '/releases/:releasePubkey/market', breadcrumb:() => <Typography variant="subtitle1">Market</Typography> },
   { path: '/collection', breadcrumb: YourCollectionBreadcrumb },
   { path: '/collection/:releasePubkey', breadcrumb: ReleaseBreadcrumb },
-  { path: '/collection/:releasePubkey/market', breadcrumb: 'Market' },
-  { path: '/upload', breadcrumb: 'Upload' },
+  { path: '/collection/:releasePubkey/market', breadcrumb: () => <Typography variant="subtitle1">Home</Typography> },
+  { path: '/upload', breadcrumb: () => <Typography variant="subtitle1">Upload</Typography> },
   { path: '/releases/:releasePubkey', breadcrumb: ReleaseBreadcrumb },
+  { path: '/:releasePubkey', breadcrumb: ReleaseBreadcrumb },
 ]
 
 const Breadcrumbs = ({ breadcrumbs }) => (
```

Comment:
the `styledReleaseBreadCrumb` is a bit jacket up, i think this was my b but we should get it fixed in this pr.

we're nesting Typography objects which is rendering `h6` as a child of a `p` which is not good. Something about the render here is passing a different `display` than the other bread crumbs and the vertical alignment is a bit off